gator.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17-alpine AS builder
+
+COPY . /tmp/gatekeeper
+
+RUN apk add --no-cache \
+    upx
+
+WORKDIR /tmp/gatekeeper/cmd/gator
+
+RUN go get -d -v
```

Comment:
We should build from the vendor folder to ensure hermiticity. Use `-mod vendor` as below:

https://github.com/open-policy-agent/gatekeeper/blob/9056cbbd77c085f2d33b4e9a29e0c084d745a00b/Dockerfile#L30

Reply:
It looks like we're still running `go get`, which shouldn't be necessary since we're pulling from the vendor directory?

Reply:
Yes, of course, I've just removed it